<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MinimumStuffSuplymant</class>
 <widget class="QDialog" name="MinimumStuffSuplymant">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>120</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>تعیین حداقل موجودی کالا در انبار</string>
  </property>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>20</y>
     <width>331</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>20</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>نام کالا:</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>136</x>
     <y>50</y>
     <width>241</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>  : </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_2"/>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>حداقل موجودی در</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>80</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>بیخیال</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>ثبت</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>comboBox</tabstop>
  <tabstop>lineEdit</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>pushButton_2</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>MinimumStuffSuplymant</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>40</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>71</x>
     <y>57</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
